statics/js/components/capture-form.js

```diff
@@ -80,17 +80,25 @@ Vue.component('capture-form', {
           <button type="button" class="btn btn-danger" @click="reset">Cancel</button>\
         </form>\
       </div>\
-      <button type="button"\
-              id="create-capture"\
-              class="btn btn-primary"\
-              v-else\
-              @click="visible = !visible">\
-        Create\
-      </button>\
+      <div v-else>\
```

Comment:
We should do the same thing for injection, I mean removing the content in case of disabled